src/jit/emitxarch.h

```diff
@@ -468,7 +472,8 @@
                                     CORINFO_SIG_INFO* sigInfo,     // used to report call sites to the EE
                                     void*           addr,
                                     ssize_t         argSize,
-                                    emitAttr        retSize,
+                                    emitAttr        retSize
```

Comment:
In emitter we should avoid UNIX_AMD64_STRUCT_PASSING #ifdef.  Instead the right macro appears to be FEATURE_MULTIREG_RET which is already defined in target.h